turbopack/crates/turbopack-tests/js/jest-entry.ts

```diff
@@ -10,6 +10,10 @@ function setupGlobals() {
   // @ts-ignore Property 'expect' does not exist on type 'typeof globalThis'
   globalThis.expect = expectMod.expect
 
+  // Set up global external values for testing
+  // @ts-ignore Property 'foo' does not exist on type 'typeof globalThis'
+  globalThis.foo = { bar: '11' }
```

Comment:
Because this is living in a different file, give this a more explicit name that's easy to grep for like `testGlobalExternalValue`